Лабораторная работа 1

Hacker228rus opened the pull request.

Hacker228rus committed:
Лабораторная работа 1

rmushkin commented on лабы 1 питон/2 задание.py:
[PEP8](https://peps.python.org/pep-0008/) (указаниям по стилю написания кода):
1. Вокруг математических операторов должны быть пробелы
2. Переменные должны иметь осмысленные имена (однобуквенные могут иметь право на существование только в исключительных случаях)

rmushkin commented on лабы 1 питон/1 задание.py:
У вас наблюдается некоторый микс решений - на второй строчке вы "хардкодите" положение пропуска (и вы в праве так делать, т. к. у вас ещё слишком мало информации из курса), но на пятой вы (тое вполне корректно) ищете пропуск автоматически. 

Выберите какой-то один подход и выполните работу целиком в его рамках.

Кроме того, [PEP8](https://peps.python.org/pep-0008/) (указания по стилю написания кода).
1. Не используйте транслит для именования переменных - Python это про красоту и однородность, а при использовании транслита код становится неоднородным, читающего вынуждают переключать контекст.
2. Окружайте математические операторы пробелами.

rmushkin reviewed (changes requested):
Поправьте, пожалуйста, файлы. Будут вопросы - пишите!

Hacker228rus committed:
Update 1 задание.py

Изменены названия переменных, поставлены пробелы вокруг математических операторов

Hacker228rus committed:
Update 2 задание.py

Поставлены пробелы между математическими операторами, однобуквенные переменные изменены на переменные из полноценных слов.

rmushkin commented on лабы 1 питон/1 задание.py:
```diff
@@ -0,0 +1,8 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+summ = sum(numbers[:4] + numbers[5:])
+lenght = len(numbers)
```
`summ` и `lenght` - это не правильные слова английского языка. Но, если что, `sum` - тоже будет плохим именем для переменной, т. к. это имя встроенной функции суммирования. Вводя такую переменную, вы лишаетесь возможности использовать функцию `sum` ниже в коде.

rmushkin reviewed (changes requested):
У вас по-прежнему некоторый микс решений (есть и хардкод, т. е. обращение к конкретному индексу, и автоматический поиск с помощью `.index`). Как я уже говорил, пожалуйста, выберите что-то одно!

P. S. И дайте мне, пожалуйста, права на запись в репозиторий, чтобы я мог принять и закрыть.
